provider/core/src/lib.rs

```diff
@@ -220,15 +219,15 @@ pub mod prelude {
     #[doc(no_inline)]
     pub use crate::DataErrorKind;
     #[doc(no_inline)]
-    pub use crate::DataKey;
+    pub use crate::DataLocale;
     #[doc(no_inline)]
-    pub use crate::DataKeyAttributes;
+    pub use crate::DataMarker;
     #[doc(no_inline)]
-    pub use crate::DataKeyHash;
+    pub use crate::DataMarkerAttributes;
     #[doc(no_inline)]
-    pub use crate::DataLocale;
+    pub use crate::DataMarkerInfo;
     #[doc(no_inline)]
-    pub use crate::DataMarker;
+    pub use crate::DataMarkerPathHash;
```

Comment:
These entries were re-ordered. For my own record:

| Old | New |
|---|---|
| DataKey | DataMarkerInfo |
| DataKeyAttributes | DataMarkerAttribute |
| DataKeyHash | DataMarkerPathHash |
| DataLocale | DataLocale |
| DataMarker | DataMarker |